UPDATING.md

```diff
@@ -52,6 +52,14 @@ assists users migrating to a new version.
 
 ## Master
 
+### `[scheduler] max_threads` config has been renamed to `[scheduler] parsing_processes`
+
+From Airflow 2.0, `max_threads` config under `[scheduler]` section has been renamed to `parsing_processes`.
+
+This is to align the name with the actual code where the Scheduler launches the number of processes defined by
+`[scheduler] parsing_processes` to Parse DAG files, calculates next DagRun date for each DAG,
+serialize them and store them in the DB.
+
```

Comment:
Should we add this to upgrade check?

Reply:
We should yeah, I will create a GH issue for it

Reply:
I think we need a new "class" of upgrade checks: 

You can make these changes now, and it will run on 1.10.13

and

You'll need to change this, but it won't work on 1.10.13.

Reply:
https://github.com/apache/airflow/issues/12672